### PR TITLE
ert speed nerf

### PR DESCRIPTION
**Pull request by ParadoxSpace**
this one's extremely simple. gives them all a slowdown of 1 except the hapt rig, which is never even seen so it's ok.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

File: code/modules/clothing/spacesuits/rig/suits/ert.dm [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	suit_type = "ERT commander"
 	icon_state = "ert_commander_rig"
 	emp_protection = 35
+	slowdown = 1
```
Comment (Sindorman) [inline code/modules/clothing/spacesuits/rig/suits/ert.dm]: This is such a terrible way to implement it. First of all you don't need lines 38, 57 and 74 as line 11 make all ert rigs have slowdown of 1 even HAPT as well, which was as you mentioned not supposed to.

Please add this override to the rig definition and add `slowdown = 0` override inside of the HAPT definition

Comment (Sindorman) [inline code/modules/clothing/spacesuits/rig/suits/ert.dm]: that's not what I asked, I want you to move this line to `/obj/item/weapon/rig/ert` definition in rig.dm file, so rigs by default have slowdown of 1.

Comment (ParadoxSpace) [reply code/modules/clothing/spacesuits/rig/suits/ert.dm]: I know what you asked. What Fowl said.

Comment (Sindorman) [reply code/modules/clothing/spacesuits/rig/suits/ert.dm]: You can't use scope of PR to justify ugliness of implementation

**Comment by Arrow768** [comment]
Closing this in favour of #6030 as it achieves the same thing and increases the code quality at the same time.

**Pull request closed without merging**